UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/HealthCheck/WebJobsAccessKeyProviderTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.Extensions.Configuration;
+using NUnit.Framework;
+using System.Collections.Generic;
+using UKHO.ExchangeSetService.Common.HealthCheck;
+
+namespace UKHO.ExchangeSetService.Common.UnitTests.HealthCheck
+{
+    public class WebJobsAccessKeyProviderTest
+    {
+        private IConfiguration configuration;
+        private WebJobsAccessKeyProvider webJobsAccessKeyProvider;
+
+        [SetUp]
+        public void Setup()
+        {
+            var inMemorySettings = new Dictionary<string, string> {
+                {"webjob1key", "webjob1value"},
+                {"webjob2key", "webjob2value"}};
+
+            configuration = new ConfigurationBuilder()
+                .AddInMemoryCollection(inMemorySettings)
+                .Build();
+
+            webJobsAccessKeyProvider = new WebJobsAccessKeyProvider(configuration);
+        }
+
+        [Test]
+        public void DoesGetWebJobAccessKeyReturnCorrectKeyWhenExists()
```

Comment:
DoesGetWebJobAccessKeyReturnCorrectKeyWhenExists

could be

GetWebJobAccessKey_ReturnsCorrectKeyWhenExists

Reply:
Fixed.